source/atlas-vector-search.txt

```diff
@@ -0,0 +1,133 @@
+.. _node-atlas-vector-search:
+
+================================
+Run an Atlas Vector Search Query
+================================
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: code example, semantic, nearest
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use the :atlas:`Atlas Vector Search
+</atlas-vector-search/vector-search-overview/>` feature
```

Comment:
```suggestion
In this guide, you can learn how to use the Atlas Vector Search feature
```
Remove the link here since there is a "learn more" at bottom of the section